smdebug/xgboost/hook.py

```diff
@@ -5,24 +5,33 @@
 
 # Third Party
 import numpy as np
+import pkg_resources
```

Comment:
Should we make sure this is installed via setup.py?

Reply:
looks `pkg_resources` comes with `setuptools` package, which is used by `setup.py`
https://setuptools.pypa.io/en/latest/pkg_resources.html
Also checked xgboost container, it has pkg_resources too, I think we don't need to install it again.